crates/ra_hir/src/code_model.rs

```diff
@@ -1075,3 +1097,13 @@ impl From<AssocItem> for crate::generics::GenericDef {
         }
     }
 }
+
+impl AssocItem {
+    pub fn module(self, db: &impl DefDatabase) -> Module {
+        match self {
+            AssocItem::Function(f) => f.module(db),
+            AssocItem::Const(c) => c.module(db),
+            AssocItem::TypeAlias(t) => t.module(db),
+        }
+    }
+}
```

Comment:
It might makes sense to land these hir and from_source additions separately, before the bulk of find usages

Reply:
Ok, but I'm not sure about `from_source` additions yet, so maybe I'll send them later.